includes/admin/settings/class-sp-settings-general.php

```diff
@@ -247,6 +247,42 @@ public function get_settings() {
 				);
 			}
 
+			$options = array_merge(
+				$options,
+				array(
+					array(
+						'type' => 'sectionend',
+						'id'   => 'general_options',
+					),
+					array(
+						'title' => esc_attr__( 'OpenStreetMap', 'sportspress' ),
+						'type'  => 'title',
+						'desc'  => '',
+						'id'    => 'osm_options',
+					),
+					array(
+						'title' => esc_attr__( 'Tile server', 'sportspress' ),
+						'id'    => 'sportspress_osm_tile_server',
+						'placeholder' => 'https://tile.openstreetmap.org/{z}/{x}/{y}.png',
+						'css'   => 'width:100%;',
+						'type'  => 'text',
+						'desc' => esc_attr__('Leave empty to use the default', 'sportspress'),
+					),
+					array(
+						'title' => esc_attr__( 'Attributions', 'sportspress' ),
+						'id'    => 'sportspress_osm_attribution',
+						'placeholder' => 'Map data &amp;copy; &lt;a href=&quot;https://www.openstreetmap.org/&quot;&gt;OpenStreetMap&lt;/a&gt; contributors',
+						'css'   => 'width:100%;',
+						'type'  => 'text',
+						'desc' => esc_attr__('Leave empty to use the default', 'sportspress'),
+					),
+					array(
+						'type' => 'sectionend',
+						'id'   => 'general_options',
+					),
+				)
+			  );
+
```

Comment:
Two minutes after posting I stumbled across the map settings on the event page which I totally missed. I think those should be moved there, but I'm not sure (I'm neither familiar with wp plugins nor this codebase) how these settings could be incorporated. Maybe something like the `other` option `sportspress_event_teams_delimiter` could work here?

I'm very open to ideas, patches or hints how this could be implemented!